relay-server/src/services/processor/span/processing.rs

```diff
@@ -219,7 +219,7 @@ struct NormalizeSpanConfig<'a> {
     /// The time at which the event was received in this Relay.
     pub received_at: DateTime<Utc>,
     /// Allowed time range for spans.
-    pub timestmap_range: std::ops::Range<UnixTimestamp>,
+    pub timestamap_range: std::ops::Range<UnixTimestamp>,
```

Comment:
LOL
```suggestion
    pub timestamp_range: std::ops::Range<UnixTimestamp>,
```

Reply:
lol I can't type, thanks for spotting it 👍